react-function-components/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click me!</button>;
+}
+
+const root = document.querySelector('#root');
+
+ReactDOM.render(CustomButton(), root);
```

Comment:
Don't call your components before passing them to the render method, let the render method choose when to call them.